api/src/Entity/Traits/Timestamps.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Entity\Traits;
+
+use DateTime;
+use DateTimeInterface;
+use Doctrine\ORM\Mapping as ORM;
+
+trait Timestamps
```

Comment:
I wonder if `Timestamps` is descriptive enough? Maybe something like `ChangeTrackingTimeStamps` so people know the intent?

Reply:
Agreed although maybe just `TrackingTimestamps`? Change to me suggest we are changing something but for the majority of these entities we are adding a tracking timestamp rather than changing because they don't exist currently

Reply:
The `CreateUpdateTimestamps` is much clearer.